common/src/main/java/com/box/l10n/mojito/okapi/filters/HtmlFilter.java

```diff
@@ -85,7 +85,7 @@ void setEmptyAndNbspAsNotTranslatable(ITextUnit textUnit) {
       String source = textUnit.getSource().toString();
       // whitespaces are collapsed by the filter so just checking for empty string & for the usage
       // of &nbsp;
-      if (source.isEmpty() || "\u00A0".equals(source)) {
+      if (source.isEmpty() || "\u00A0".equals(source) || !textUnit.getSource().hasText()) {
```

Comment:
may be able to remove `source.isEmpty()` at least test are passing but not 100%. keeping it like this for now